spell/spell-crafter-goerli-workflow.md

```diff
@@ -98,6 +98,7 @@ PR: https://github.com/makerdao/spells-goerli/pull/TODO
 * [ ] Pre-Deploy Setup and Checks (currently via `dapptools`)
   * [ ] Set local env (`.sethrc`)
     * [ ] Deployer
+      * [ ] Avoid using the same deployer for mainnet and testnet(to avoid deploying contracts with the same address but different sources)
```

Comment:
Needs a space

`testnet(to` --> `testnet (to`

Weird that spell check didn't catch this though!

Reply:
18cf91c